less/mixins.less

```diff
@@ -1 +1,17 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
```

Comment:
try creating mixings for simple things like flexbox or padding so you don't have to keep typing them out.